### ISSUE TITLE
Training command

### ISSUE THREAD

**Issue by abhigoku10** [euphoria0-0/Learning-Loss-for-Active-Learning#1]
@euphoria0-0  thanks for sharing the code base i have following query 
1. the command "python main.py --task detection --dataset VOC0712 \
    --subset None --num_epoch 300 --batch_size 32 \
    --lr 0.001 --epoch_loss 240 --weights 1.0 --milestone " is incomplete with subset and milestone value not given can you confirm 
2. Dataset voc0712 ? should have both voc2007 and voc2012 in our repo 

Thanks in advance

**Comment by euphoria0-0** [comment]
@abhigoku10 sorry for the delay.
1. For object detection task, subset is not defined and the milestone should be 240. I will revise the milestone value in the README command. In the paper, the subset was not used for object detection task. 
```We do not create a random subset $\mathcal{S}_M$ since the size of the pool $\mathcal{U}_{16,551}$ is not very large in contrast to CIFAR-10.```
2. Yes. In the paper, the authors mentioned that ```we make a super-set trainval’07+12 by combining the two.```

Thanks a lot your questions. Any further ones are welcomed.

**Comment by abhigoku10** [comment]
@euphoria0-0  thanks for the response i have below queries 
1. when subset is NONE i get error that it should be an int since in the args "'--subset', type=int, default=10000, help='subset for learning loss'" its mentioned like this 
2. how many hours does it take to train object detection or kpe tasks? 
3. for validation of the data after a sample it loads voc.pth model file does it  update after every trails ?
Thanks in advance

**Comment by euphoria0-0** [comment]
@abhigoku10 Thank you for your questions again.
1. I've just edited the default value of ```--subset``` argument. Thanks!
2. I'm not sure but it was about 1 day at the final round of active learning with a single machine with RTX 3090 in my case.
3. Yes. If you need the previous file (before updating), you can just modify the file to save.
Thanks a lot.

**Comment by abhigoku10** [comment]
@euphoria0-0  thanks for the response currently i have put if for training on V100 with the updated code 
1. I was facing issues with HPE module also wit the command can you please double check it once 

Thanks in advance

**Comment by euphoria0-0** [comment]
@abhigoku10 
The command is working well, but I found the training issue that the default hyper-parameter value is different from the original paper. ```--wdecay``` should be ```0``` for HPE.
I corrected the code, and you can add ```--wdecay 0``` in your command.
Thanks.

**Comment by abhigoku10** [comment]
@euphoria0-0  hi thanks for the response , currently i am training on the V100 machine single one with th esame settings as mentioend but still only Round 6/10 is completed from 4 days for default dataset and the mAP is not changing
OC07 metric? Yes
AP for aeroplane = 0.5818
AP for bicycle = 0.6337
AP for bird = 0.3976
AP for boat = 0.4045
AP for bottle = 0.1938
AP for bus = 0.6745
AP for car = 0.6923
AP for cat = 0.6230
AP for chair = 0.2706
AP for cow = 0.5076
AP for diningtable = 0.4891
AP for dog = 0.5744
AP for horse = 0.6862
AP for motorbike = 0.7041
AP for person = 0.6186
AP for pottedplant = 0.2154
AP for sheep = 0.4486
AP for sofa = 0.5460
AP for train = 0.6490
AP for tvmonitor = 0.5529
Mean AP = 0.5232
~~~~~~~~
Results:
0.582
0.634
0.398
0.404
0.194
0.674

Any thoughts on this

**Comment by euphoria0-0** [comment]
@abhigoku10 Thank you for letting me know. I'll check.
